corehq/apps/accounting/tests/test_forms.py

```diff
@@ -12,6 +12,15 @@
 )
 from corehq.apps.accounting.tests.test_invoicing import BaseInvoiceTestCase
 
+from corehq.apps.accounting.tests.base_tests import BaseAccountingTest
```

Comment:
It's recommended to keep imports organized.  Different files follow different conventions, but often are similar to:

Python built-in imports

Third party library imports

HQ imports

When in doubt, I try to follow whatever pattern is already in use by the file.